.jenkins/common.groovy

```diff
@@ -74,28 +133,20 @@ def runTestCommand (platform, project) {
                     cd ../ && mkdir -p conformance && cd conformance
                     pip3 install pandas
                     mkdir hevc-conformance && cd hevc-conformance
-                    wget http://math-ci.amd.com/userContent/computer-vision/HevcConformance/*zip*/HevcConformance.zip
-                    unzip HevcConformance.zip
-                    python3 /opt/rocm/share/rocdecode/test/testScripts/run_rocDecode_Conformance.py --videodecode_exe ./../../rocdecode-sample/videodecode --files_directory ./HevcConformance --results_directory .
+                    python3 /opt/rocm/share/rocdecode/test/testScripts/run_rocDecode_Conformance.py --videodecode_exe ./../../rocdecode-sample/videodecode --files_directory \${JENKINS_HOME_DIR}/rocDecode/HevcConformance --results_directory .
                     cd ../
                     mkdir avc-conformance && cd avc-conformance
-                    wget http://math-ci.amd.com/userContent/computer-vision/rocDecodeConformance/AvcConformance.zip
-                    unzip AvcConformance.zip
-                    python3 /opt/rocm/share/rocdecode/test/testScripts/run_rocDecode_Conformance.py --videodecode_exe ./../../rocdecode-sample/videodecode --files_directory ./AvcConformance --results_directory .
+                    python3 /opt/rocm/share/rocdecode/test/testScripts/run_rocDecode_Conformance.py --videodecode_exe ./../../rocdecode-sample/videodecode --files_directory \${JENKINS_HOME_DIR}/rocDecode//AvcConformance --results_directory .
                     cd ../
                     mkdir vp9-conformance && cd vp9-conformance
-                    wget http://math-ci.amd.com/userContent/computer-vision/rocDecodeConformance/Vp9Conformance.zip
-                    unzip Vp9Conformance.zip
-                    python3 /opt/rocm/share/rocdecode/test/testScripts/run_rocDecode_Conformance.py --videodecode_exe ./../../rocdecode-sample/videodecode --files_directory ./Vp9Conformance --results_directory .
+                    python3 /opt/rocm/share/rocdecode/test/testScripts/run_rocDecode_Conformance.py --videodecode_exe ./../../rocdecode-sample/videodecode --files_directory \${JENKINS_HOME_DIR}/rocDecode//Vp9Conformance --results_directory .
                     cd ../
                     mkdir av1-conformance && cd av1-conformance
-                    wget http://math-ci.amd.com/userContent/computer-vision/rocDecodeConformance/Av1Conformance_v1.0.zip
-                    unzip Av1Conformance_v1.0.zip
-                    python3 /opt/rocm/share/rocdecode/test/testScripts/run_rocDecode_Conformance.py --videodecode_exe ./../../rocdecode-sample/videodecode --files_directory ./Av1Conformance_v1.0 --results_directory .
+                    python3 /opt/rocm/share/rocdecode/test/testScripts/run_rocDecode_Conformance.py --videodecode_exe ./../../rocdecode-sample/videodecode --files_directory \${JENKINS_HOME_DIR}/rocDecode//Av1Conformance_v1.0 --results_directory .
```

Comment:
Remove the extra slash after rocDecode: `rocDecode//Av1Conformance_v1.0`  ->` /rocDecode/Av1Conformance_v1.0`